### ISSUE TITLE
ImportError: No module named cefpython3

### ISSUE THREAD

**Issue by MrEngineer13** [cztomczak/cefpython#227]
I am new to Kivy, so I wan't entirely sure what repo I should've added this issue to. I want to add a windowed browser to my Kivy app. To do so, the only way I found to do it is with [CEF Python](https://github.com/cztomczak/cefpython)

Then I run the [Kivy example here](https://github.com/cztomczak/cefpython/blob/master/src/linux/binaries_64bit/kivy_.py) and I keep getting this error:

> File "/Users/User/Documents/Kivy/browser.py", line 28, in <module>
>     from cefpython3 import cefpython
> ImportError: No module named cefpython3

And I've already installed cefpython3 like so
`pip install cefpython3`

> Requirement already satisfied (use --upgrade to upgrade): cefpython3 in /Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages

But when I try to install cefpython3 I get this error.
`kivy -m pip install cefpython3`

> Downloading/unpacking cefpython3
>   Could not find any downloads that satisfy the requirement cefpython3
> Cleaning up...
> No distributions at all found for cefpython3
> Storing debug log for failure in /Users/User/.pip/pip.log

I am using Python 2.7.11 on my Mac running OSX. How can I resolve this issue?

**Comment by cztomczak** [comment]
Go to the location where cefpython3 package is installed and delete the files. Try upgrading/downgrading pip or use the setup package to install cefpython3: https://github.com/cztomczak/cefpython/releases/download/v31.2/cefpython3-31.2-macosx-setup.tar.gz

**Comment by cztomczak** [comment]
What OS X and python are you running (the default system python?) ? 

I looked at other packages at PyPI and their whl package files have "cp27-cp27m" in its filename. In cefpython3 there is "cp27-none" and I wonder if this could have something to do with the issue. The "cp27m" is a python version tag, but it's not explained in PEP 425 what does the "m" suffix mean.

**Comment by MrEngineer13** [comment]
I'm on OSX 10.11.4, here's the version of Python I'm using (I downloaded it from Python.org) - 
Python 2.7.11 (v2.7.11:6d1b6a68f775, Dec  5 2015, 12:54:16) 
[GCC 4.2.1 (Apple Inc. build 5666) (dot 3)] on darwin

When I tried to run the setup package, I got the error below. All I did was download via the link you provided, extracted it and tried to run the script like so:

`python /Users/User/Downloads/cefpython3-31.2-macosx-setup\ 4/setup.py`

This was the error I got: 

> Traceback (most recent call last):
>   File "/Users/User/Downloads/cefpython3-31.2-macosx-setup 4/setup.py", line 77, in <module>
>     + get_package_data("Resources/", "cefpython3/")
>   File "/Users/User/Downloads/cefpython3-31.2-macosx-setup 4/setup.py", line 35, in get_package_data
>     os.chdir(base_dir)
> OSError: [Errno 2] No such file or directory: 'cefpython3/

So I changed the file permissions and then installed cefpython3 and this error has been resolved. I am experiencing a new one, so to keep this issue clean and on topic, I'll just open another issue.
